src/main/java/edu/ucla/library/iiif/auth/delegate/Config.java

```diff
@@ -63,24 +73,27 @@ public Config() {
 
         myCookieService = getURI(Config.AUTH_COOKIE_SERVICE);
         myTokenService = getURI(Config.AUTH_TOKEN_SERVICE);
-        myAccessService = getURI(Config.AUTH_ACCESS_SERVICE);
+        mySinaiTokenService = getURI(Config.SINAI_AUTH_TOKEN_SERVICE);
+        myAccessModeService = getURI(Config.AUTH_ACCESS_MODE_SERVICE);
```

Comment:
I'm a little iffy on propagating Mode here too. I'm not sure it's adding value, and it's lengthening the variable names.

Reply:
I think the change is an improvement, but I'm fine with reverting